yoti/yoti.module

```diff
@@ -310,11 +311,6 @@ function yoti_user_view($account, $view_mode, $langcode) {
       }
     }
 
-    // Format date of birth.
```

Comment:
Are we just getting rid of this because it's being done in `processProfileAttributes`?

Reply:
Yes, it will be already formatted when it gets in here.